testdata/certs/README.md

```diff
@@ -0,0 +1 @@
+This empty directory will be used by telemetry_test.go to add certs to.
```

Comment:
>empty [](http://example.com/codeflow?start=5&length=5)

Your solution works. However, there is a community fancier implementation: [https://stackoverflow.com/a/932982](https://stackoverflow.com/a/932982)

Reply:
Done.